pkg/apis/istio/v1alpha2/deepcopy.go

```diff
@@ -15,22 +15,22 @@
 package v1alpha2
 
 import (
+	"encoding/json"
 	"github.com/gogo/protobuf/proto"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 
-	"istio.io/operator/pkg/util"
 	"istio.io/pkg/log"
 )
 
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
```

Comment:
If this is autogenerated do we need to change the generation code instead of directly here? although it doesn't mention where its generated from..

Reply:
Autogenerated deepcopy is usually found in `zz_generated.deepcopy.gen.go`. Looks like the code (and this comment) here was copied from one of those generated files, but is now maintained manually. I may be wrong, as I've only just begun looking at the code here.

Reply:
It's not autogenerated, but I don't think it will work - see comment below.